chord_metadata_service/patients/api_views.py

```diff
@@ -149,34 +159,46 @@ class IndividualBatchViewSet(BatchViewSet):
     def get_queryset(self):
         individual_ids = self.request.data.get("id", None)
         filter_by_id = {"id__in": individual_ids} if individual_ids else {}
-        queryset = Individual.objects.filter(**filter_by_id)\
-            .prefetch_related(
-                *(f"phenopackets__{p}" for p in PHENOPACKET_PREFETCH if p != "subject"),
-        ).order_by("id")
+        queryset = (
+            Individual.objects
+            .filter(**filter_by_id)
+            .prefetch_related(*(f"phenopackets__{p}" for p in PHENOPACKET_PREFETCH if p != "subject"))
+            .order_by("id")
+        )
 
         return queryset
 
 
-async def public_discovery_filter_queryset(request: DrfRequest, queryset: QuerySet, low_counts_censored: bool):
-    # Check query parameters validity
-    qp = request.query_params
-    discovery = await get_request_discovery(request)
-    # TODO: allow exceeding max query parameters for authorized requests
-    if len(qp) > get_max_query_parameters(discovery, low_counts_censored):
+async def public_discovery_filter_queryset(
+    discovery: DiscoveryConfig, request: DrfRequest, dt_permissions: DataTypeDiscoveryPermissions, queryset: QuerySet
+):
+    # Process query parameters and check validity
+
+    qp: QueryDict = deepcopy(request.query_params)
+
+    # - remove project/dataset (i.e., scope) query parameters
```

Comment:
ah was this messing up the fields count?

Reply:
yes, exactly - i'll expand on that a bit in the comment